Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
```

Comment:
The use of multiple commands in the 'dev_server' task can lead to readability and maintainability issues. Consider breaking this down into smaller, more manageable scripts or functions within a dedicated script file to streamline the execution process and enhance clarity.